src/helpers/helpers.dom.js

```diff
@@ -39,11 +39,14 @@ function parseMaxStyle(styleValue, node, parentProperty) {
  * @param {HTMLElement} domNode - the node to check the constraint on
  * @param {string} maxStyle - the style that defines the maximum for the direction we are using ('max-width' / 'max-height')
  * @param {string} percentageProperty - property of parent to use when calculating width as a percentage
- * @return {number|undefined} number or undefined if no constraint
+ * @return {number=} number or undefined if no constraint
  * @see {@link https://www.nathanaeljones.com/blog/2013/reading-max-width-cross-browser}
  */
 function getConstraintDimension(domNode, maxStyle, percentageProperty) {
 	var view = document.defaultView;
+	if (!view) {
```

Comment:
I think its going to be easier to debug without this check, when the view is not defined.

Reply:
I'm not sure how you think this should be handled. Should we throw an error instead? That's basically what would happen before. But we can't leave the code exactly as it was before and enable the strict checks because we need to handle `null` and `undefined` cases explicitly